roles/databases/influxdb/defaults/main.yaml

```diff
@@ -0,0 +1,9 @@
+dojot_influxdb_version: v2.0.2
+dojot_influxdb_port: 8086
+dojot_influxdb_admin_user: dojot
```

Comment:
I don't know if * _admin_ * is the correct term, it can be confused with org.

Reply:
That was changed.